experimental/Schemes/BlowupMorphism.jl

```diff
@@ -96,108 +97,223 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   return Z_trans, inc_Z_trans, pr_res
 end
 
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient wrapper to the MPolyIdeal-method
```

Comment:
Please put it where it belongs.

Reply:
done

Reply:
Yes, I did not push yet, because the strict-transform-fix slowed K3.jl down and I am currently profiling whether this is inevitable or due to a bad way of fixing this.